webapp/models.py

```diff
@@ -26,7 +28,19 @@ class Asset(Base):
     __tablename__ = "asset"
 
     id = Column(Integer, primary_key=True)
-    created = Column(DateTime, nullable=False)
+    author = Column(String, nullable=True)
+    title = Column(String, nullable=True)
+    asset_type = Column(String, nullable=True)
+    product = Column(String, nullable=True)
```

Comment:
this field is very similar to Tags fields, so it should have a similar data type

Reply:
shouldn't this change from "String" to "relationship"?

Reply:
Yes, I've updated it